Storage comparison documentation (#78)
Certora/Documentation: docs/cvl/types.md

sometimes called hyperproperties.

CVL supports this kind of specification using the special `storage` type.  A

> variable of type `storage` represents a snapshot of the EVM storage, as well
> as shadow state declared using ghosts, at a given point in time.

Reviewer: ```suggestion
variable of type `storage` represents a snapshot of the EVM storage and the state of ghosts at a given point in time.
```
